pkg/apis/ceph.rook.io/v1/nfs.go

```diff
@@ -51,19 +50,15 @@ func (n *CephNFS) IsHostNetwork(c *ClusterSpec) bool {
 	return c.Network.IsHost()
 }
 
-func (n *CephNFS) ValidateCreate() (admission.Warnings, error) {
+func (n *CephNFS) ValidateCreate() (error, error) {
 	return n.Spec.Security.Validate()
 }
 
-func (n *CephNFS) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
+func (n *CephNFS) ValidateUpdate(old runtime.Object) (error, error) {
```

Comment:
Why two errors are returned? That's very unusual.

Reply:
updated code